config/locales/views.es.yml

```
@@ -169,6 +169,27 @@ es:
       create:
         from_comment: 'De comentario por %{user} el %{date}'
 
+  licenses:
+    general_payment_reason: 'Pago mensual MawidaBP'
+    trial_valid_until: 'Período prueba valida hasta'
+    price_per_month: 'Precio por mes'
+    statuses:
+      unpaid: 'Impago'
+      active: 'Activo'
+      trial: 'Prueba'
+      cancelled: 'Cancelado'
+    blocked:
+      show:
+        body: 'Su organización está inhabilitada por falta de pago'
```

```suggestion
        body: 'Su organización está inhabilitada, contacte sopore@mawidabp.com'
```

Así no somos tan _secos_. Debería ir con `_html` y un link tipo "mailto:", pero me dio fiaca escribirlo =P

---

Acá mas que contacte, debería ser "por favor realice el pago " sino te van a mandar mail, y es porque los paja no pagaron.

---

Con la cantidad _prevista_ que tenemos de muchachos "online" no creo que lleguemos a ese problema. Si llegamos lo cambiamos, lo prometo =P